tools/deployment_tool/test/deployment_tools.test.js

```diff
@@ -0,0 +1,171 @@
+
+            EIP20Token.setAddress('0x0000000000444440000000000000000000000100');
+
+            const registry = new ContractRegistry();
+            registry.addContract(EIP20Token);
+
+            const output = registry.toParityGenesisAccounts();
+            assert(Object.keys(output).includes('0x0000000000444440000000000000000000000100'));
+            assert.equal(Object.keys(output).length, 1);
+        });
+
+        it('can not set address after instantiation', () => {
+            const constructorAbi = EIP20TokenFile.abi.find(n => n.type === 'constructor');
+            const EIP20Token = new Contract(EIP20TokenFile.contractName, EIP20TokenFile.bytecode, constructorAbi, ['MYT', 'MyToken', 18]);
+
+            EIP20Token.setAddress('0x0000000000444440000000000000000000000100');
+            EIP20Token.instantiate();
+
+            assert.throws(() => EIP20Token.setAddress('0x0000000000444440000000000000000000000100'));
+        });
+    });
+
+    describe('.loadTruffleContract()', () => {
+        it('loads correct contract when provided with rootDir option', () => {
+            const EIP20Token = Contract.loadTruffleContract(
+                'EIP20Token',
+                ['MYT', 'MyToken', 18],
+                { rootDir: `${__dirname}/../../../` },
+            );
+
+            assert.equal(EIP20Token.contractName, 'EIP20Token');
+        });
+    });
+});
+
+
+describe('ContractRegistry', () => {
+    describe('toParityGenesisAccounts', () => {
+        it('linking works for 2 contracts', () => {
+            const MerklePatriciaProof = Contract.loadTruffleContract('MerklePatriciaProof', null, { rootDir });
+            const MessageBus = Contract.loadTruffleContract('MessageBus', null, { rootDir });
+            MessageBus.addLinkedDependency(MerklePatriciaProof);
+
+            const registry = new ContractRegistry();
+            registry.addContracts([MerklePatriciaProof, MessageBus]);
+
+            const output = registry.toParityGenesisAccounts();
+            assert.equal(Object.keys(output).length, 2);
+            // check that link placeholders have been removed
+            Object.values(output).forEach(bytecode => assert.isNotOk(bytecode.constructor.includes('_')));
+        });
+
+        it('orders contracts correctly (correct order provided)', () => {
+            const MerklePatriciaProof = Contract.loadTruffleContract('MerklePatriciaProof', null, { rootDir });
+            const MessageBus = Contract.loadTruffleContract('MessageBus', null, { rootDir });
+            MessageBus.addLinkedDependency(MerklePatriciaProof);
+
+            // provided ordering = correct ordering
+            const registry = new ContractRegistry();
+            registry.addContracts([MerklePatriciaProof, MessageBus]);
+
+            const output = registry.toParityGenesisAccounts();
+            assert.equal(Object.keys(output)[0], MerklePatriciaProof.getAddress());
```

Comment:
We generally use strictEqual for the asserts.
Assert reason/messages are missing in all the assert statements.